packages/flutter/lib/src/widgets/single_child_scroll_view.dart

```diff
@@ -42,11 +43,17 @@ class SingleChildScrollView extends StatelessWidget {
     this.scrollDirection: Axis.vertical,
     this.reverse: false,
     this.padding,
+    this.primary: false,
```

Comment:
Side-note: our widget constructors need dartdoc (I was gonna say this param should be documented, only to realize the whole constructor needs to be documented :-))